atlas/packages/app/src/shared/components/VideoPlayer/VideoPlayer.style.tsx

```diff
@@ -134,7 +135,9 @@ export const PlayOverlay = styled.div`
   cursor: pointer;
 `
 
-export const StyledPlayIcon = styled(PlayIcon)`
+const StyledIcon = styled(Icon)`
   height: 72px;
   width: 72px;
+  color: ${colors.white};
 `
+export const StyledPlayIcon = ({ ...svgProps }) => <StyledIcon name="play" {...svgProps} />
```

Comment:
FYI, I _think_ you should be able to do this inline with defining the styled component, like so:

```tsx
const StyledIcon = styled(({ ...props }) => <Icon name="play" {...props} />)`
  height: 72px;
  width: 72px;
  color: ${colors.white};
`
```

Your solution is perfectly fine as well so don't bother changing it

Reply:
good catch! Will keep it in mind for the next time something like this comes out!